Tests/UITests/ResponseStubs/tooltip-toucharea.json

```diff
@@ -0,0 +1,54 @@
+{
+    "currentPingMillis": 0,
+    "nextPingMillis": 3600000,
+    "data": [
+        {
+            "campaignData": {
+                "campaignId": "t1",
+                "maxImpressions": 999,
+                "type": 2,
+                "isTest": false,
+                "isCampaignDismissable": true,
+                "hasNoEndDate": true,
+                "infiniteImpressions": true,
+                "triggers": [
+                    {
+                        "type": 1,
+                        "eventType": 1,
+                        "eventName": "app_start",
+                        "attributes": []
+                    }
+                ],
+                "messagePayload": {
+                    "title": "[Tooltip] tooltip 1",
+                    "messageBody": "{\"UIElement\" : \"tooltip.test.1\", \"position\": \"top-left\", \"auto-disappear\": 3, \"redirectURL\": \"https://www.google.com\"}",
```

Comment:
Why do we need **\"auto-disappear\": 3** in `tooltip-toucharea.json`,` for testing this tooltip and don't in `tooltip-data.json`?

Reply:
"auto-disappear is not required in tooltip-toucharea.json, will remove it

Reply:
Is it possible to use single json file (with a single tooltip) for touch area testing and other tests that use `tooltip-data.json`?